src/App.tsx

```diff
@@ -1,9 +1,28 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function getFilteredMovies(query: string) {
+  const loweredQuery = query.toLowerCase().trim();
+
+  const filteredMovies = moviesFromServer.filter((film) => (
+    film.title.toLowerCase().includes(loweredQuery)
+    || film.description.toLowerCase().includes(loweredQuery)
+  ));
```

Comment:
Let's improve readability a bit

```suggestion
  const filteredMovies = moviesFromServer.filter(movie => {
    const movieTitle = film.title.toLowerCase();
    const movieDesc = film.description.toLowerCase();
    
    return movieTitle.includes(loweredQuery)
    || movieDesc.includes(loweredQuery)
  });
```